frontend/packages/console-app/src/components/quick-starts/QuickStartPanelContent.tsx

```diff
@@ -27,18 +29,19 @@ const QuickStartPanelContent: React.FC<QuickStartPanelContentProps> = ({
   handleClose,
   activeQuickStartID,
 }) => {
-  const [scrollDirection, handleScrollCallback] = useScrollDirection();
   const { t } = useTranslation();
+  const [shadows, SetShadows] = React.useState<Shadows>(Shadows.none);
```

Comment:
use camelCasing for names
```suggestion
  const [shadows, setShadows] = React.useState<Shadows>(Shadows.none);
```

Reply:
updated